server/components/users/user-controllers.js

```diff
@@ -0,0 +1,43 @@
+const User = require("./user-model");
+
+exports.getAllUsers = async (req, res) => {
+	try {
+		const users = await User.find({});
```

Comment:
The amount of users returned here should be limited, as done in [the Post component](https://github.com/R-Layer/Bears-Team-22/blob/feature/post-component/server/components/posts/post-controllers.js#L5-L9).